dart/lib/flat_buffers.dart

```diff
@@ -28,13 +28,15 @@ typedef void StructBuilder();
 class BufferContext {
   final ByteData _buffer;
 
+  ByteData get buffer => _buffer;
+
   factory BufferContext.fromBytes(List<int> byteList) {
     Uint8List uint8List = _asUint8List(byteList);
     ByteData buf = new ByteData.view(uint8List.buffer, uint8List.offsetInBytes);
-    return new BufferContext._(buf);
+    return BufferContext(buf);
   }
 
-  BufferContext._(this._buffer);
+  BufferContext(this._buffer);
```

Comment:
Why are we making this public?

If we need it to be public we should document it.

Reply:
Hmm, that slipped in while I've been comparing/merging with the null-safe version from [objectbox](https://github.com/objectbox/objectbox-dart/) - it's used there for performance reasons - no need for list of bytes to construct if `ByteData` is the actually needed interface.

So while this is unrelated to the current PR, I'd leave it here as it doesn't hurt anyone and actually provides a bit of value for lib users. Added a doc comment.

Reply:
SG, docs haven't shown up here yet though (forgot to save/push?)

Reply:
it was there and pushed, not sure why it doesn't show up https://github.com/google/flatbuffers/pull/6696/commits/304056a7f6860003137bd4170580ede763862d56